pkgs/build-support/rust/default-crate-overrides.nix

```
@@ -13,15 +13,15 @@ in
   cargo = attrs: {
     buildInputs = [ openssl zlib curl ]
       ++ stdenv.lib.optionals stdenv.isDarwin [ CoreFoundation libiconv ];
```

It looks like my macOS build error can be fixed with

```suggestion
      ++ stdenv.lib.optionals stdenv.isDarwin [ CoreFoundation Security libiconv ];
```

Of course, then I get the same error for cargo-vendor itself, so the same fix needs to be applied there too.